src/irmin/store_intf.ml

```diff
@@ -429,8 +429,14 @@ module type S = sig
         operations as the generating computation, in the exact same order.*in
         some order*. *)
 
-    val produce_stream : (stream_proof, 'a) producer
-    (** [produce_stream] is the producer of stream proofs. *)
+    val produce_stream :
+      ?with_extenders:bool ->
```

Comment:
could you simplify this even further and always assume that `with_extenders` is `true`?

Or if for some reason you want to keep this (not sure why :p) you could write: 
```ocaml
val produce_stream :  ?with_extenders:bool -> (stream_proof, 'a) producer
```

Reply:
removed the `with_extenders` argument